cluster-up/cluster/kind-ovn/provider.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash -ex
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2024 Red Hat, Inc.
+#
+
+KIND_VERSION=0.19.0
+export KIND_IMAGE=kindest/node
+export K8S_VERSION=v1.28.0@sha256:dad5a6238c5e41d7cac405fae3b5eda2ad1de6f1190fa8bfc64ff5bb86173213
+
+KIND_PATH=${KIND_PATH:-"${KUBEVIRTCI_CONFIG_PATH}/${KUBEVIRT_PROVIDER}/_kind"}
+CLUSTER_PATH=${CLUSTER_PATH:-"${KUBEVIRTCI_CONFIG_PATH}/${KUBEVIRT_PROVIDER}/_ovnk"}
+
+MTU=${MTU:-"1392"}
```

Comment:
Let's try to calculate it taking the MTU of the interface from the default gw route

```bash
overlay_overhead=58
current_mtu=$(cat /sys/class/net/$(ip route | grep "default via" | head -1 | awk '{print $5}')/mtu)
overlay_mtu=$(expr $current_mtu - $overlay_overhead)
```

Reply:
but why ? there is no need, one can just export MTU in case he has smaller MTU
and it is rare
we can just add readme instead
and make the code simpler

Reply:
If something changes on the prow infra, your code will break, you won't know why, you'll have to debug, spend time on it, eventually figure out the MTU changed, and update one magic value by another magic value.

Quique's suggestion will work under those circumstances. 

> it is rare

? How so ? Why do you think this is strange ?

Reply:
> but why ? there is no need, one can just export MTU in case he has smaller MTU and it is rare we can just add readme instead and make the code simpler

@oshoval by default this should try to calculate it with something like the bash snippet and also give users a scape hatch in case the calculations are wrong.

Reply:
> ? How so ? Why do you think this is strange ?

based on the machines i saw, usually MTU is not that small
anyhow i will do that